.circleci/config.yml

```diff
@@ -111,18 +86,8 @@ jobs:
       - run:
           name: Build a distributable package
           command: |
-            source venv/bin/activate
-            # Build a wheel release
-            if [[ $CIRCLE_TAG ]]; then
-                # This is a tagged release
-                inv pydist
-            elif [[ "$CIRCLE_BRANCH" == feature/* ]]; then
-                # This is a feature branch
-                inv pydist -b $CIRCLE_BUILD_NUM+${CIRCLE_BRANCH#*/}
-            else
-                # This is a simple development build
-                inv pydist -b $CIRCLE_BUILD_NUM
-            fi
+            poetry version prerelease
```

Comment:
This doesn't push back to git -> we would always bump to the same prerelease version?